### PR TITLE
Iterate.MountainDuck version 4.7.1.18339

### PR DESCRIPTION
**Pull request by chausner**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25097)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
(refs/pull/25097/head): PR validation failed. Actual: 3010, Expected: 0. Please test your WinGet package inside of a virtual machine. For more infomation about error codes, please read https://docs.microsoft.com/en-us/windows/win32/msi/error-codes.

**Comment by chausner** [comment]
> (refs/pull/25097/head): PR validation failed. Actual: 3010, Expected: 0. Please test your WinGet package inside of a virtual machine. For more infomation about error codes, please read https://docs.microsoft.com/en-us/windows/win32/msi/error-codes.

Exit code 3010 means success but a reboot is required. I have fixed it in the manifest.

**Review by OfficialEsco — COMMENTED** [review]
Uhhh is there no NORESTART switch for this? It just instantly rebooted without any warning which is a very bad user experience...

File: manifests/i/Iterate/MountainDuck/4.7.1.18339/Iterate.MountainDuck.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Iterate.MountainDuck
+PackageVersion: 4.7.1.18339
+Installers:
+- Architecture: x64
+  InstallerType: burn
+  InstallerUrl: https://dist.mountainduck.io/Mountain%20Duck%20Installer-4.7.1.18339.exe
+  InstallerSha256: 24DED835D1A36C259A9B25A7B500074C68D88B65B41A2FA0AD56E58CE7D51496
```
Comment (OfficialEsco) [inline manifests/i/Iterate/MountainDuck/4.7.1.18339/Iterate.MountainDuck.installer.yaml:10]:
```suggestion
  InstallerSha256: 24DED835D1A36C259A9B25A7B500074C68D88B65B41A2FA0AD56E58CE7D51496
  InstallerSwitches:
    Custom: /norestart
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
99% sure /norestart worked

**Comment by chausner** [comment]
> 99% sure /norestart worked

Ah great, thanks!

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**